README.md

```diff
@@ -79,40 +79,48 @@ Download the key and then create a Base64 string from it. It will be required la
 
 ## Secret Key, Webhook Token and Integration ID
 
-You'll need the right private key to run the Integration. It is not supplied here. `Versionbot` requires the following environment variables:
+You'll need the right private key to run the Integration. It is not supplied here. `ProcBot` requires the following environment variables:
 
     `WEBHOOK_SECRET`: The 20 digit hex key used to authenticate messages.
     `INTEGRATION_ID`: The ID given on Integration creation, a unique identifier.
     `PROCBOTS_PEM`: The Base64 encoded private key generated on Integration creation.
+	`FLOWDOCK_ALERTS`: If present and set to 'true', alerts will be sent to any relevant Flowdock room.
+
+`VersionBot` requires the following environment variables:
+
 	`VERSIONBOT_NAME`: The name shown in commits and merges for PRs by the Integration.
 	`VERSIONBOT_EMAIL`: Email address for the bot, (can be an empty string).
+	`VERSIONBOT_FLOWDOCK_ROOM`: The room ID whose inbox will be posted to. If `FLOWDOCK_ALERTS` is set to `true` this **must** be present.
```

Comment:
Yeah, I'd ummed and ahhed about this; I've been thinking about something else that would affect this, but in retrospect it's probably a bad idea.